tools/run-container

```diff
@@ -463,9 +463,12 @@ main() {
         return
     }
 
-    inside_as_cd "$name" root "$cdir" \
-        python3 ./tools/read-dependencies "--distro=${OS_NAME}" \
-            --test-distro || {
+    local rdcmd=(python3 tools/read-dependencies "--distro=${OS_NAME}" --install --test-distro)
+    if [[ ${OS_NAME} == centos ]]; then
+        error "WARNING: Unable to install all CentOS for test dependencies (missing deps, LP: #1867151)."
+        unittest=""
+    fi
```

Comment:
We should be able to drop this and allow running unittests for centos because tip has now dropped our dependency on both unittest2 and contextlib2.

I was able to pass 
/tools/run-container --source-package --unittest --artifacts=./srpm/ centos/8
...
=================== 2306 passed, 10 skipped in 54.97 seconds ===================

with this diff 
```diff
diff --git a/packages/pkg-deps.json b/packages/pkg-deps.json
index 114e921e4..800283966 100644
--- a/packages/pkg-deps.json
+++ b/packages/pkg-deps.json
@@ -17,10 +17,6 @@
       "build-requires" : [
          "python3-devel"
       ],
-      "renames" : {
-         "unittest2" : "OMIT: LP: #1867151 Missing CentOS pkg deps",
-         "contextlib2" : "OMIT: LP: #1867151 Missing CentOS pkg deps"
-      },
       "requires" : [
          "e2fsprogs",
          "iproute",
diff --git a/tools/run-container b/tools/run-container
index 31a39b069..588f09234 100755
--- a/tools/run-container
+++ b/tools/run-container
@@ -466,7 +466,6 @@ main() {
     local rdcmd=(python3 tools/read-dependencies "--distro=${OS_NAME}" --install --test-distro)
     if [[ ${OS_NAME} == centos ]]; then
         error "WARNING: Unable to install all CentOS for test dependencies (missing deps, LP: #1867151)."
-        unittest=""
     fi
     inside_as_cd "$name" root "$cdir" "${rdcmd[@]}" || {
         errorrc "FAIL: failed to install dependencies with read-dependencies"
```